app/assets/v2/js/pages/fulfill_bounty/index.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-console */
-if (bountyChainId !== '58') {
+if (bountyChainId === '1') {
```

Comment:
why are we changing this. ?

Reply:
it limits web3 modal to show for only eth while submitting work. it wrongly pops-up for every chain... previously I exempted binance chain (58)

Reply:
@octavioamu yea i think that'll be better... pushing a commit now